src/api/app/models/bs_request.rb

```diff
@@ -53,6 +59,7 @@ def set_ignore_build_state
   end
 
   def skip_sanitize
+    'ñlsdakjfsñadljfñasldjfñasldkjf sdañlkjf lñasdkjf ñasdlkjf lñaskdjf ñlaskdjf lñksdajf ñlsadjf ñlaskjdf ñlkasdjf ñlasdkjf ñlkasdj fñlaskdj fñlkasdj fñlkasdj fñlkjasd ñlfjkasd ñlfkjasd ñlfjk asdlñkjf asñdlkjf ñlasdkjf ñlakjsd fñlkasjd flkñasj dñlkasd jfñlkasjd ñfljasd ñlfjasd lñkfjas lñksaf jdlñas'
```

Comment:
Line is too long. [301/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.